package.json

```diff
@@ -77,7 +77,7 @@
     "cheerio": "^1.0.0-rc.2",
     "easyimage": "2.1.0",
     "escomplex": "^2.0.0-alpha",
-    "eslint": "^6.8.0",
+    "eslint": "^4.18.2",
```

Comment:
Is the up version a problem?
You should check if the problem is solved compared to the previous one.

Reply:
@TomaszLukawskiSam 
need to check.

Reply:
This changes is not matched with the title..
'From 3.19.0 to 4.18.2'  :(

Reply:
@lslachciak @DohyungLim 
I don't see purpose of this PR. I suggest closing this PR